lib/fedex/tracking_information.rb

```diff
@@ -29,7 +29,7 @@ class TrackingInformation
     }
 
     attr_reader :tracking_number, :signature_name, :service_type, :status,
-                :delivery_at, :events, :unique_tracking_number
+                :delivery_at, :events, :unique_tracking_number, :estimated_delivery_at
```

Comment:
I don't actually see `estimated_delivery_at` used anywhere in the fashionthing PR. It's using `estimated_delivery_timestamp`